documentation/deployment-manual.md

```diff
@@ -0,0 +1,180 @@
+# Deploy the IoT Offline Dashboarding sample manually
+
+This document describes how to deploy the IoT Offline Dashboarding sample manually on an Edge device. Chose this option to gain a deep understanding of every required step involved in deploying the sample.
+
+**Table of contents**
+* [Preparation of Edge boxes and IoT Hub](#preparation-of-edge-boxes-and-iot-hub)
+* [Building all Edge module images](#building-all-edge-module-images)
+* [Deploying all Edge modules](#deploying-all-edge-modules)
+* [See also](#see-also)
+
+## Preparation of Edge boxes and IoT Hub
+
+**Please start by setting up your Edge devices by going through the [edge environment setup](setup-edge-environment.md) document.**
+
+## Building all Edge module images
+
+Before any Edge modules can be deployed, it is necessary to build the module images using the Dockerfiles found in the repository. Once built, the images need to be placed into a container registry.
+
+Start by cloning [the github repository](https://github.com/AzureIoTGBB/iot-edge-offline-dashboarding) to your local machine.
+
+```bash
+git clone https://github.com/AzureIoTGBB/iot-edge-offline-dashboarding.git
+```
+
+Replace `{registry}` in the commands below with the container registry location on your machine (e.g. myregistry.azurecr.io).
```

Comment:
the registry location is not "on your machine"...  it's generally going to be in azure..   maybe say "the container registry location created earlier" or something like that

Reply:
Done. See commit [0212d04088f5955795e04005bb2a2cedb1bd989b](https://github.com/DirkSonguer/iot-edge-offline-dashboarding/commit/0212d04088f5955795e04005bb2a2cedb1bd989b).